### ISSUE TITLE
bpython can't be installed

### ISSUE THREAD

**Issue by fdev31** [rubik/pyg#110]
```
fab@fab-desktop:/tmp% sudo pyg -d install -U bpython                                                                                14h08
Loading options from /home/fab/.pyg/pyg.conf
Looking for bpython releases on PyPI
Best match: bpython==0.10.1
Downloading bpython [100% - 81.4 Kb / 81.4 Kb]              
Checking md5 sum                                
Running setup.py egg_info for bpython
Traceback (most recent call last):
  File "/usr/local/bin/pyg", line 9, in <module>
    load_entry_point('pyg==0.8a', 'console_scripts', 'pyg')()
  File "/home/fab/progs/pyg/pyg/__init__.py", line 68, in main
    parser.dispatch(argv=argv)
  File "/usr/local/lib/python2.7/site-packages/argh-0.14.0-py2.7.egg/argh/helpers.py", line 304, in dispatch
    return dispatch(self, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/argh-0.14.0-py2.7.egg/argh/helpers.py", line 205, in dispatch
    for line in lines:
  File "/usr/local/lib/python2.7/site-packages/argh-0.14.0-py2.7.egg/argh/helpers.py", line 281, in _execute_command
    for line in result:
  File "/usr/local/lib/python2.7/site-packages/argh-0.14.0-py2.7.egg/argh/helpers.py", line 264, in _call
    result = args.function(args)
  File "/home/fab/progs/pyg/pyg/parser/parser.py", line 100, in install
    args_manager['install']['ignore'])
  File "/home/fab/progs/pyg/pyg/parser/_opts.py", line 94, in install_func
    _install_package_from_name(package, ignore)
  File "/home/fab/progs/pyg/pyg/parser/_opts.py", line 74, in _install_package_from_name
    return Installer(package).install()
  File "/home/fab/progs/pyg/pyg/inst.py", line 93, in install
    r.install()
  File "/home/fab/progs/pyg/pyg/req.py", line 151, in install
    self._download_and_install(url, name, p.name, pext, hash)
  File "/home/fab/progs/pyg/pyg/req.py", line 131, in _download_and_install
    installer.install()
  File "/home/fab/progs/pyg/pyg/core.py", line 283, in install
    Dir(os.path.dirname(setup_py), self.name, tempdir, self.reqset).install()
  File "/home/fab/progs/pyg/pyg/core.py", line 228, in install
    for r in dist.requires['install']:
AttributeError: 'Dir' object has no attribute 'requires'
fab@fab-desktop:/tmp%                                                                                                               14h08
```

**Comment by fdev31** [comment]
I have that issue with most packages, am I the only one ? :P

**Comment by fdev31** [comment]
Another issue I didn't see before: if I installed a package with python setup.py develop, then I remove it with pyg it will totally remove the sources folder....
I don't think this is the expected behavior, we should filter results according to "standard" paths only and ask if there is additional files/folders detected....

**Comment by rubik** [comment]
Sorry, I have changed the Dist object in pkgtools. :) Now all Dist objects have a requires attribute that holds install requirements and extras (before pkgtools couldn't detect extras). So I have released pkgtools 0.7.1. Sorry! Here are the commits:
https://github.com/rubik/pkgtools/commit/173eb7b51fb5b467bb6c7b769f6cbd338b59bae1
https://github.com/rubik/pkgtools/commit/d901fa8fb6fab6f7f06988bf33ea484e16f59331

and Pyg's one:
https://github.com/rubik/pyg/commit/31df47a07f9f014138ce9192633d97ecf9db223d

As for the other issue, we should ask the user when he wants to remove a develop package.
